src/screens/Landing/components/Categories.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { BESTSELLERS, RECOMMENDED } from "../../../data/landingPageData";
+import { BookList } from "./BookList";
+import { Link } from "react-router-dom";
+
+export const Categories = () => {
+	return (
+		<>
+			<h1 className="main-header text-center">Discover Your Next Book</h1>
+			<div className="main-content">
+				<h2 className="main-content-header text-center">BESTSELLERS</h2>
```

Comment:
having all caps text on your site is bad for accessibility. people find it hard to read.
https://github.com/humanmade/hm-pattern-library/issues/75

Reply:
Hmm...I've never come across this ever...I'll change this

Reply:
Actually, I'll come back to this later, it kind of looks odd without the caps